swap_meet/vendor.py

```diff
@@ -1,2 +1,63 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory or []
+
+    def add(self, added_item):
+        (self.inventory).append(added_item)
```

Comment:
[nit] `self.inventory` doesn't have to be in parens

```suggestion
        self.inventory.append(added_item)
```